stories/Components/UIcomponents/Hero/PageHero/PageHeroOption.jsx

```diff
@@ -19,28 +19,33 @@ export function PageHeroOption({
   cta,
   ...args
 }) {
-  const CtaEnabledClass = args.CTA == 'On' ? 'has-cta' : '';
+  const CtaEnabledClass = args.CTA == "On" ? "has-cta" : "";
   return (
-    <div className={['pagehero-tall', CtaEnabledClass].join(' ')}>
+    <div className={["pagehero-tall", CtaEnabledClass].join(" ")}>
       <div className="grid-x">
-        <div className="cell medium-4 medium-offset-1">
+        <div className="cell medium-4 medium-offset-1 rtl-offset-medium-1">
```

Comment:
No need for the new class, please check the comment for the _base.css and remove it. Same for all other affected components